PluginsMakefile.mk

```
@@ -103,7 +115,14 @@ npm: ## Run a npm command, example: make npm c='install mypackage/package'
 .PHONY: npm
 
 ##—— Testing and static analysis ———————————————————————————————————————————————
-verify: license-headers-check parallel-lint phpstan phpcsfixer-check phpunit ## Run all our lints/tests/static analysis
+verify:  ## Run all our lints/tests/static analysis
+	@$(call run_always, license-headers-check)
```

To be compliant with the CI workflow. People may want to create a plugin with the MIT license and not have a license header in all their file, but just a license file in the root dir of their plugin.

```suggestion
	@$(call run_if_exists, tools/HEADER, license-headers-check)
```